### PR TITLE
6821 validate max breadcrumb config setting

### PR DESCRIPTION
**Pull request by rich-bugsnag**
## Goal

We need to check that the config value max breadcrumbs is set to within 0 - 100

## Changeset

- Added a check to the setter of Configuration.MaxBreadcrumbs
- Added a range slider to the bugsnag gameobject script 

## Testing

Added unit tests and manually tested.

NOTE: Also added Maze-Runner tests for the MaxBreadcrumb setting

### CODE REVIEW

File: test/mobile/features/ios/ios_enabledbreadcrumbs.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Feature: android enabled breadcrumbs
```
Comment (kattrali) [inline test/mobile/features/ios/ios_enabledbreadcrumbs.feature:1]:
```suggestion
Feature: ios enabled breadcrumbs
```

?

Why have separate files for enabled vs max breadcrumbs? Seems reasonable to have one `breadcrumbs.feature` for anything related.

Comment (rich-bugsnag) [reply test/mobile/features/ios/ios_enabledbreadcrumbs.feature]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,61 +4,59 @@
...
+* Added event metadata for CPU and graphics capabilities and migrated entries from the Unity tab to the device and app tabs, to better match other platforms ([#297](https://github.com/bugsnag/bugsnag-unity/pull/297)):
+  * `unity.companyName` -> `app.companyName`
+  * `unity.productName` -> `app.name`
+  * `unity.version` -> `app.version`
+  * `unity.platform` -> `app.type`
+  * `unity.osLanguage` -> `device.osLanguage`
+  * `unity.unityException` -> removed as a duplicate of the error class
+  * `unity.unityLogType` -> removed as is contained in the error class for generic logs
+	
+* Added advanced config option 'EnabledBreadcrumbTypes' to enable/disable automatically recorded breadcrumbs ([#301](https://github.com/bugsnag/bugsnag-unity/pull/301))
```
Comment (tomlongridge) [inline CHANGELOG.md:16]:
```suggestion
* Add `EnabledBreadcrumbTypes` configuration option to enable/disable automatically recorded breadcrumbs [#301](https://github.com/bugsnag/bugsnag-unity/pull/301)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,61 +4,59 @@
 
 ### Enhancements
 
+* Added event metadata for CPU and graphics capabilities and migrated entries from the Unity tab to the device and app tabs, to better match other platforms ([#297](https://github.com/bugsnag/bugsnag-unity/pull/297)):
```
Comment (tomlongridge) [inline CHANGELOG.md:7]:
```suggestion
* Add event metadata for CPU and graphics capabilities and migrated entries from the Unity tab to the device and app tabs, to better match other platforms ([#297](https://github.com/bugsnag/bugsnag-unity/pull/297)):
```

**Review by tomlongridge — COMMENTED** [review]
✅  Changelog

**Review by kattrali — APPROVED** [review]

**Pull request merged**